src/web/data/userMapData.json

```diff
@@ -1,9 +1,10 @@
 {
-  "lastUpdated": "End of June 2018",
+  "lastUpdated": "Middle of March 2019",
```

Comment:
Let this be Feb 2019, since we don't have the data from March 2019, it would be inaccurate to say this is correct as of March.

Reply:
Done changed it to Mid February 2019